JeffShepherd/Rancid-Tomatillos
# Refactor/configure api calls

## What does this PR do?

- Refactors application to use data returned via API call instead of using test data stored within the application.

## How should it be tested in the UI?

- cd into Rancid-Tomatillos repo, run npm start
- On load, the page should be populated with all movie
- Upon clicking a specific movie, the movie details view should display, and include all details for that specific movie
- To test error handling for the API returning all movies, enter 'http://rancid-tomatillos.herokuapp.com/api/v2/mov' for the relevant fetch URL. An error message should render right below the header.
- To test error handling for the API returning specific movie details, enter 'http://rancid-tomatillos.herokuapp.com/api/v2/movies/12' for the relevant fetch URL. An error message should render right below the header.
